azkaban-common/src/main/java/azkaban/project/Project.java

```diff
@@ -496,6 +516,30 @@ public void setVersion(final int version) {
     this.version = version;
   }
 
+  public void isCRTLocked(boolean flag) {
```

Comment:
Looks like this is a setter - can we rename it to sth like `setCRTLocked`?

Reply:
It was setCRTLocked in the my first version, but I feel like set something locked == pass this value to true. Will setCRTLock(true/false) instead setCRTLocked makes more sense to you?

Reply:
Yup `setCRTLock` looks good to me